package ru.turikhay.util.async;

class AsyncObjectException extends Exception {
   AsyncObjectException() {
   }

   AsyncObjectException(Throwable cause) {
      super(cause);
   }
}
